.ddev/config.yaml

```diff
@@ -0,0 +1,267 @@
+type: drupal
+docroot: web
+php_version: "8.3"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
```

Comment:
router_http_port and router_https_port are not generally used in project config.yaml any more, instead done globally.

```suggestion
```